eng/pipelines/templates/stages/archetype-c-release.yml

```
@@ -0,0 +1,92 @@
+                        Get-Content $(Build.SourcesDirectory)/eng/common/create-tags-and-git-release.ps1 | Write-Host
+                      displayName: create-tags-and-git-release.ps1 contents
+                    - pwsh: >-
+                          $(Build.SourcesDirectory)/eng/common/create-tags-and-git-release.ps1
+                          -artifactLocation "$(Pipeline.Workspace)/packages/${{artifact.Name}}"
+                          -packageRepository C
+                          -releaseSha $(Build.SourceVersion)
+                          -repoId $(Build.Repository.Name)
+                      displayName: 'Verify Package Tags and Create Git Releases'
+                      timeoutInMinutes: 5
+                      env:
+                        GH_TOKEN: $(azuresdk-github-pat)
+
+          - ${{if ne(artifact.options.skipPublishDocs, 'true')}}:
+            - deployment: PublishDocs
+              displayName: Publish Docs to GitHub pages
+              condition: ne(variables['Skip.PublishDocs'], 'true')
+              environment: githubio
+
+              pool:
+                vmImage: windows-2019
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: none
+                      - pwsh: Write-Host 'publish docs'
+                      - pwsh: |
+                          Get-ChildItem -Recurse $(Pipeline.Workspace)/docs/${{ artifact.Name }}
+                        displayName: Output Visible Artifacts
+                      - template: $(Build.SourcesDirectory)/eng/common/pipelines/templates/steps/publish-blobs.yml
+                        parameters:
+                          FolderForUpload: '$(Pipeline.Workspace)/docs/${{ artifact.Name }}'
+                          BlobSASKey: '$(azure-sdk-docs-prod-sas)'
+                          BlobName: '$(azure-sdk-docs-prod-blob-name)'
+                          TargetLanguage: 'c'
+                          # we override the regular script path because we have cloned the build tools repo as a separate artifact.
+                          ScriptPath: '$(Build.SourcesDirectory)/eng/common/scripts/copy-docs-to-blobstorage.ps1'
+
+
+          - ${{if ne(artifact.options.skipUpdateSdkVersion, 'true')}}:
+            - deployment: UpdateSdkVersion
+              displayName: "Update SDK Version"
+              condition: and(succeeded(), ne(variables['Skip.UpdateSdkVersion'], 'true'))
+              environment: github
+
+              pool:
+                vmImage: windows-2019
+
+              strategy:
+                runOnce:
+                  deploy:
+                    steps:
+                      - checkout: self
+                      - pwsh: |
+                          eng/Update-SdkVersion.ps1
+                        displayName: Increment SDK version
+                      - template:$(Build.SourcesDirectory)/eng/common/pipelines/templates/steps/create-pull-request.yml
```

This may not be valid YAML

```suggestion
                      - template: $(Build.SourcesDirectory)/eng/common/pipelines/templates/steps/create-pull-request.yml
```

---

I believe you can also just go `/eng/common/pipelines/templates/steps/create-pull-request.yml`